spec/api/v1/requests/image_request_spec.rb

```diff
@@ -95,6 +95,23 @@
   end
 
   describe 'delete' do
-    it 'can delete an image'
+    it 'can delete an image' do
```

Comment:
Don't know if I will have time to implement this, but thankyou for building it out anyway!